project_generator/exporters/uvision_definitions.py

```diff
@@ -119,12 +119,14 @@ def get_mcu_definition(self, name):
             }
         },
     }
+    # alias mbed standard names to their CPUs:
+    mcu_def['K64F'] = mcu_def['MK64FN1M0xxx12']
```

Comment:
There're board definitions, which can be used. I created this week an issue on mbed to rename targets to more sane names. K64F is frdm-k64f platform. If platforms in mbed have proper naming, we can reuse them in board definitions (k64f -> frmd-k64f, lpc1768 -> mbed-lpc1768, etc...), otherwise we might need a translation table as it's above. Ideas?